glutz/types.go

```diff
@@ -12,7 +12,7 @@ type DeviceDb struct {
 	AccessPoint      string   `json:"accessPoint"`
 	OperatingMode    int64   `json:"operatingMode"`
 	Firmware         string  `json:"firmware"`
-	OpenableDuration string  `json:"openableDuration"` // Check this!
+	Openable 		 bool  `json:"openable"` 
```

Comment:
Boolean is the right type, but in Eliona the output button sets the output data from 0 to 1 if clicked. So it is better to change the type from bollean to int. But the int value has to be interpreted like an boolean.